zcash_primitives/Cargo.toml

```diff
@@ -96,7 +96,6 @@ orchard = { version = "0.4", default-features = false, features = ["test-depende
 
 [target.'cfg(unix)'.dev-dependencies]
 pprof = { version = "0.11", features = ["criterion", "flamegraph"] } # MSRV 1.56
```

Comment:
```suggestion
pprof = { version = "0.11", features = ["criterion", "flamegraph"] }
```

Reply:
I'd prefer to leave this in as it's useful for documenting the MSRV of a dependency that has caused us problems before.